packages/dashboard-backend/src/routes/api/devworkspaceResources.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2018-2023 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import { FastifyInstance, FastifyRequest } from 'fastify';
+import * as axios from 'axios';
+import * as https from 'https';
+import { baseApiPath } from '../../constants/config';
+import { getSchema } from '../../services/helpers';
+import { prebuiltResourcesSchema } from '../../constants/schemas';
+import { restParams } from '../../typings/models';
+import { dump } from 'js-yaml';
+import { Main as DevworkspaceGenerator } from '@eclipse-che/che-devworkspace-generator/lib/main';
+import { V1alpha2DevWorkspaceTemplate } from '@devfile/api';
+import { certificateAuthority } from './helpers/getCertificateAuthority';
+
+const tags = ['Devfile'];
```

Comment:
Let's keep it consistent:

```suggestion
const tags = ['DevWorkspace Resources'];
```

Reply:
I have fixed it